pcsx2-qt/Settings/InterfaceSettingsWidget.cpp

```diff
@@ -173,8 +173,13 @@ InterfaceSettingsWidget::InterfaceSettingsWidget(SettingsDialog* dialog, QWidget
 	dialog->registerWidgetHelp(
 		m_ui.discordPresence, tr("Enable Discord Presence"), tr("Unchecked"),
 		tr("Shows the game you are currently playing as part of your profile in Discord."));
-	dialog->registerWidgetHelp(m_ui.doubleClickTogglesFullscreen, tr("Double-Click Toggles Fullscreen"), tr("Checked"), tr(""));
-	dialog->registerWidgetHelp(m_ui.disableWindowResizing, tr("Disable Window Resizing"), tr("Unchecked"), tr(""));
+	dialog->registerWidgetHelp(
+		m_ui.doubleClickTogglesFullscreen, tr("Double-Click Toggles Fullscreen"), tr("Checked"), 
+		tr("Whilst you are on the in-game screen, you can double-click anywhere on the screen to go in and out of fullscreen mode. "
```

Comment:
I'd avoid having specific hotkeys mentioned here, because the user can change them. Probably best to keep it simple, something along the lines of `Allows switching in and out of fullscreen mode by double-clicking the game window.`